src/domain/ecoverse/ecoverse.service.ts

```diff
@@ -292,31 +292,54 @@ export class EcoverseService {
 
   // Create the user and add the user into the members group
   async createUser(userData: UserInput): Promise<IUser> {
+    const user = await this.createUserProfile(userData);
+    if (this.accountService.authenticationEnabled()) {
+      await this.createUserAccount(user.id);
+    }
+    return user;
+  }
+
+  // Create the user and add the user into the members group
+  async createUserAccount(userID: number): Promise<boolean> {
```

Comment:
I don't believe this is Ecoverse-specific logic and I don't think this should be in the Ecoverse service.

Reply:
Creating of a user is an ecoverse action, as it is admitting a user in - creating an account could be on a resolver on the account module?